packages/cli/src/WorkflowRunner.ts

```diff
@@ -125,18 +125,17 @@ export class WorkflowRunner {
 		// does contain those messages.
 		try {
 			// Search for messages for this executionId in event logs
-			const { eventBus } = await import('./eventbus');
+			const { MessageEventBus } = await import('./eventbus');
```

Comment:
Does it make sense to lazy load it? Since the event bus is so intertwined into our code? Might as well just add it to the top alongside the rest of the imports wdyt?

Also we cannot use `import('@/eventbus')` right? Since it's runtime.